scripts/install

```diff
@@ -15,5 +15,6 @@ else
     PIP="pip"
 fi
 
+"$PIP" install -U "pip >= 20.2" setuptools wheel
 "$PIP" install -r "$REQUIREMENTS"
 "$PIP" install -e .
```

Comment:
We can remove this line since `-e .[http2]` has been included in requirements.txt .

Reply:
Good point, though it's actually the case not even considering this PR :-)

```suggestion
```